src/tests/Interop/COM/ComWrappers/GlobalInstance/GlobalInstance.Marshalling.BuiltInComDisabled.cs

```diff
@@ -0,0 +1,53 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace ComWrappersTests.GlobalInstance
+{
+    using System;
+
+    using ComWrappersTests.Common;
+    using TestLibrary;
+
+    partial class Program
+    {
+        private static void ValidateNotRegisteredForTrackerSupport()
+        {
+            Console.WriteLine($"Running {nameof(ValidateNotRegisteredForTrackerSupport)}...");
+
+            int hr = MockReferenceTrackerRuntime.Trigger_NotifyEndOfReferenceTrackingOnThread();
+            Assert.AreNotEqual(GlobalComWrappers.ReleaseObjectsCallAck, hr);
+        }
+
+        static int Main(string[] doNotUse)
```

Comment:
Instead of having a separate copy like this, could we re-use `GlobalInstance.Marshalling.cs`, making it get the feature switch value and skip certain validation based on if built-in COM support is disabled?